MAINTAINERS.md

```diff
@@ -0,0 +1,199 @@
+    - [`priority/important-longterm`](#priorityimportant-longterm)
+    - [`priority/backlog`](#prioritybacklog)
+  - [Step 3: Assign the Work](#step-3-assign-the-work)
+  - [Step 4: Follow up](#step-4-follow-up)
+- [Release process](#release-process)
+
+
+## Issue Triage
+
+<details>
+<summary>Source</summary>
+The Issue Triage section of the guide was heavily inspired by and occasionally uses the excellent <a href="https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md">Kubernetes Contributors Guide</a>.
+</details>
+
+**Maintainers are encouraged to regularly triage incoming issues and PRs**, which means:
+- A short response to the issue author acknowledging that the issue was seen
+- Evaluating and labelling the issue according to type, priority and urgency
+- Assigning the issue/PR to a contributor or maintainer for work or review
+
+Triage can happen asynchronously and continuously, or in regularly scheduled meetings.
+
+**Benefits of triaging are:**
+- Maintaining project momentum
+- Engagement of contributors by increasing responsiveness
+- Prevents piling up of issues and work
+- Prevents issues and the project itself becoming stale
+- Provides structure and an opportunity for collaborative decision-making with regards to the project trajectory and roadmap
+
+It can be beneficial to have triage be part of already existing regular meetings, and if the volume of incoming issues requires it, have a regular, public triage meeting. If there is a regular meeting, make sure it is added here and in the [CONTRIBUTING.md](/CONTRIBUTING.md). Also consider adding it to the [NEW-CONTRIBUTORS.md](/NEW-CONTRIBUTORS.MD) and explicitly invite new and prospective contributors to listen in or participate. This will give interested people a plannable, social opportunity to meet the maintainers and gain insight into the project, with both a low barrier to entry for the new contributors and a low amount of additional effort for the maintainers.
+
+### Step 1: Type the Issue
+
+Issues can be of various types, some of which have labels:
+
+#### `Support` for support requests
+
+Support requests, eg. issues that are not bugs and exclusively concern the usage of the code as it is, will be converted in to a discussion and closed.
+
+Here is [an example](https://github.com/openpgpjs/openpgpjs/issues/1661) of this occuring
+
+If you find yourself doing this a lot, your `.github/ISSUE_TEMPLATE` needs improvement. The template should be effectively guiding people in need of support to the discussions and away from the issues.
+
+If you realise that the question is a duplicate of an existing discussion, and you can easily find that discussion, it’s worth linking to the existing one instead of creating a new one.
+
+If questions are frequently asked, they might be a frequently asked question! Consider adding the solution to the documentation, Wiki or FAQ, whichever applies and is easily discoverable.
+
```

Comment:
These should go in Discussions instead. Also, the link 404s, which I guess is good (it might've been moved to Discussions :upside_down_face:)
```suggestion
```

Reply:
IMO this could fit here as part of the "guidelines" idea mentioned in https://github.com/openpgpjs/openpgpjs/pull/1711#discussion_r1444374718 🙂 

In fact, we do have some Wiki entries we'd been wanting to write (e.g. about streaming), and a I think we have a few candidate questions for a FAQ as well (e.g. session key decryption errors).